packages/solana/lib/src/rpc/dto/account_data/spl_token_program/mint_account_data_info.dart

```diff
@@ -0,0 +1,23 @@
+import 'package:json_annotation/json_annotation.dart';
+
+part 'mint_account_data_info.g.dart';
+
+@JsonSerializable()
```

Comment:
`toJson` is not used, should we remove it from generation? Same for other classes, I think for responses it doesn't make sense to keep it.